src/site/content/en/blog/storage-for-the-web/index.md

```diff
@@ -65,9 +66,7 @@ They are synchronous, and are not accessible from web workers. Like
 LocalStorage and SessionStorage, cookies are limited to only strings.
 
 The [File System API][mdn-fileapi] and FileWriter API provide methods for
-reading and writing files to a sandboxed file system. While it is asynchronous,
-it is not recommended because it is
-[only available in Chromium-based browsers][caniuse-fs].
```

Comment:
Please keep this.

Reply:
The information for this on caniuse is out of date. Here's [MDN's data](https://developer.mozilla.org/en-US/docs/Web/API/File_and_Directory_Entries_API/Introduction#browser_compatibility).

Reply:
I don't trust that and want to verify this myself before accepting that. I'll try to give it a shot tomorrow.

Reply:
Let me know when you have an answer.